tools/topology/platform/intel/byt.m4

```diff
@@ -34,3 +34,9 @@ VIRTUAL_WIDGET(codec_out0, output, 1)
 VIRTUAL_WIDGET(codec_out1, output, 2)
 VIRTUAL_WIDGET(codec_in0, input, 3)
 VIRTUAL_WIDGET(codec_in1, input, 4)
+VIRTUAL_WIDGET(ssp2 Rx, out_drv, 5)
+VIRTUAL_WIDGET(ssp2 Tx, out_drv, 6)
+VIRTUAL_WIDGET(ssp0 Tx, out_drv, 7)
+VIRTUAL_WIDGET(ssp0 Rx, out_drv, 8)
+VIRTUAL_WIDGET(modem_out, out_drv, 9)
+VIRTUAL_WIDGET(modem_in, input, 10)
```

Comment:
this works, but shouldn't we clean the out_drv and input/output to be consistent?

Reply:
@plbossart is it better now?

Reply:
Dunno. what is the opposite of out_drv? plain vanilla input?

Reply:
OK fixed now